wolfssl/ssl.h

```diff
@@ -3735,6 +3735,16 @@ WOLFSSL_API void *wolfSSL_OPENSSL_memdup(const void *data,
 WOLFSSL_API void wolfSSL_ERR_load_BIO_strings(void);
 #endif
 
+#ifdef HAVE_OCSP
```

Comment:
Might also consider wrapping this in `NO_ASN_TIME`... some embedded targets will not be happy.

Reply:
Nice catch!  Changed to
```
#if defined(HAVE_OCSP) && !defined(NO_ASN_TIME)
```
there and in ssl.c to gate the definitions themselves.